demo/WebexMeetingWidgetDemo.jsx

```diff
@@ -0,0 +1,46 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {Input} from '@momentum-ui/react';
+
+import {WebexMeetingWidget} from '../src';
+
+function WebexMeetingWidgetDemo({token}) {
+  const [destination, setDestination] = useState('');
+  const [displayWidget, setDisplayWidget] = useState(false);
+
+  const displayButtonEnabled = token && destination && !displayWidget;
+
+  return (
+    <div>
```

Comment:
Maybe a fragment is a better choice here
```suggestion
    <React.Fragment>
```